sklearn/metrics/_plot/det_curve.py

```diff
@@ -1,14 +1,15 @@
 # Authors: The scikit-learn developers
 # SPDX-License-Identifier: BSD-3-Clause
 
+import numpy as np
 import scipy as sp
 
 from ...utils._plotting import _BinaryClassifierCurveDisplayMixin
 from .._ranking import det_curve
 
 
 class DetCurveDisplay(_BinaryClassifierCurveDisplayMixin):
-    """DET curve visualization.
+    """Detection Error Tradeoff (DET) curve visualization.
```

Comment:
Same as `def det_curve`: Could you fix the below link to the user guide to https://scikit-learn.org/stable/modules/model_evaluation.html#detection-error-tradeoff-det?

Reply:
Done for `DetCurveDisplay` and it's methods. The cross-references to the user guide in in the RocCurveDisplay and in PrecisionRecallDisplay point to https://scikit-learn.org/stable/visualizations.html, shall we change those as well in another PR?